### PR TITLE
Markdown and reference fixes

### PR DESCRIPTION
**Pull request by jfbastien**

### CODE REVIEW

File: cpp20_synchronization_library.bs [hunk 1]
```diff
@@ -88,9 +88,9 @@ This paper is the unification of the wording for a series of related C++20
 proposals for introducing new synchronization and thread coordination
 facilities and enhancing existing ones:
 
-- [[P0514]]: Efficient `atomic` waiting and semaphores.
-- [[P0666]]: Latches and barriers.
-- [[P0995]]: `atomic_flag::test` and lockfree integral types.
+  - [[P0514r4]]: Efficient `atomic` waiting and semaphores.
+  - [[P0666r2]]: Latches and barriers.
+  - [[P0995r1]]: `atomic_flag::test` and lockfree integral types.
```
Comment (brycelelbach) [inline cpp20_synchronization_library.bs:93]: I don't understand this diff? Why the spaces before hand? Just for prettiness in the source?

**Review by brycelelbach — APPROVED** [review]
See my question about the list formatting (I've got no issue with it, just curious).

File: cpp20_synchronization_library.bs (hunk 1 above)
Comment (jfbastien) [reply cpp20_synchronization_library.bs:93]: You're supposed to indent lists, and have blank lines before / after them. Bikeshed was erroring out a bunch before my fixes.

Comment (brycelelbach) [reply cpp20_synchronization_library.bs:93]: I didn't think indentation was needed, I've never gotten warnings about that.

**Pull request merged**